.github/workflows/msbuild.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         useVcpkgToolchainFile: true
         buildWithCMake: true
         buildDirectory: ${{ env.buildDir }}
-        cmakeAppendedArgs: '-GNinja'
+        cmakeBuildType: ${{env.BUILD_CONFIGURATION}}
 
```

Comment:
Release build is easier for users to test, so +1 to  that. I suppose BUILD_CONFIGURATION defaults to Release.
I do not know what -GNinja means, but you own this and you know better.

Reply:
- BUILD_CONFIGURATION is defined at the top of this script in "env" block as "Release"
- -GNinja tells CMake to generate Ninja build system script, which is now set by default, so I removed it. An alternative to Ninja could be, for example, msvc. See https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html#id9